pkgs/tools/misc/beam-book/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib, stdenv
+, fetchFromGitHub
+, wget, asciidoctor, ditaa, graphviz
+, gitMinimal
+, erlang
+, jdk
+, rsync
+, rouge
+}:
+
+let
+  pname = "the-beam-book";
+  version = "0.0.14.fix";
+
+  bookDirName = "${pname}-${version}";
+
+  book = fetchFromGitHub {
+    name = bookDirName;
```

Comment:
```suggestion
```

Reply:
I need a name to refer to this source by. When I use `sourceRoot = book`, that fails because that is read-only in the store. How else should I refer to both the main book source and the supplementary file I need to grab?